ssl/statem/statem.c

```diff
@@ -174,8 +175,7 @@ int ossl_statem_accept(SSL *s)
     return state_machine(s, 1);
 }
 
-static void (*get_callback(SSL *s))(const SSL *, int, int)
-{
+static void (*get_callback(SSL *s)) (const SSL *, int, int) {
```

Comment:
Yeah I noticed this one. I think it's fine to let it make rare style mistakes (so long as it makes them less often than a human...) but I can indent-off if you prefer.